conf/messages

```diff
@@ -894,6 +894,14 @@ benefits.beneficialLoans.theseAre.agent = These are any interest free or low int
 benefits.beneficialLoans.error.noEntry.individual = Select yes if you got beneficial loans
 benefits.beneficialLoans.error.noEntry.agent = Select yes if your client got beneficial loans
 
+benefits.educational-ServicesEmployment.title.individual = Did you get any benefits for educational services?
+benefits.educational-ServicesEmployment.title.agent = Did your client get any benefits for educational services?
+benefits.educational-ServicesEmployment.theseAre.individual = These are scholarships or school fees paid for by your employer.
```

Comment:
It seams there is no test for the text

Reply:
The tests are under the render the page text with title checks etc